console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -959,11 +978,21 @@ const fetchViewDefinition = (viewName, isRedirect) => {
   };
 };
 
-const deleteViewSql = viewName => {
+const deleteViewSql = (viewName, viewType) => {
   return (dispatch, getState) => {
     const currentSchema = getState().tables.currentSchema;
+    const property = viewType.toLowerCase();
+    const capitalizedProperty = property[0].toUpperCase() + property.slice(1);
```

Comment:
Could you add it to `jsUtils` as `capitalize` function?

Reply:
Done! Fixed the few other call sites I could find.